CommonControls/PackFileBrowser/PackFileBrowserViewModel.cs

```diff
@@ -36,11 +37,25 @@ public TreeNode SelectedItem
             }
         }
 
+        public class DragDoneParameters
```

Comment:
Is there not already a drop parameter somewhere? Its possible to drag from the tree view into the kitbash editor so I am pretty sure I made something at some point?

Reply:
Can the other kitbash editor use this new class instead? Just trying to avoid multiple classes that does almost the same

Reply:
The treeview should probably implement IDropTarget so the two behave the same?